### DISCUSSION TITLE
Unable to install next-app

### DISCUSSION THREAD

**Discussion by kunalnhes** [vercel/next.js#14552] [Help] [answered]
Running init next-app is throwing error
-------------------------------------------------------------------------------------------------------------------------------------
D:\React\NextTut>npm init next-app nextjs-blog
npm ERR! code ENOLOCAL
npm ERR! Could not install from "k\AppData\Roaming\npm-cache\_npx\10804" as it does not contain a package.json file.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Kunal k\AppData\Roaming\npm-cache\_logs\2020-06-25T07_46_55_991Z-debug.log
Install for [ 'create-next-app@latest' ] failed with code 1
---------------------------------------------------------------------------------------------------------------------------------------
OS: Windows
NodeJs version: v12.16.1
NPM Version: 6.13.4
------------------------------------------------------------------------------------------------------------------------------
I tried changing the npm cache path from "Kunal k" to  "Kunal~1" as depicted by the command "DIR /x" but no luck.
Tried verifying and force cleaning npm cache no luck.

It seems the issue is with the space in my user name but not sure about it.

Please Help me solve this I am new to next don't understand much about the error. FYI create-react-app works perfectly  fine without any issue.

**Comment by jamesmosier** [comment]
I think you may be right that there are issues with a space in your directory: https://github.com/npm/init-package-json/issues/72

I think internally `npm init` is the same as `npx`, but does `npx create-next-app` result in the same issue?

**Reply by kunalnhes** [reply]
Yes, it throws the same error. I did get a workaround for the issue, would like to share it for future reference I have shared [here](https://github.com/vercel/next.js/discussions/14428#discussioncomment-30169)

**Reply by igor-olikh** [reply]
Did you find a solution?

**Reply by osaroDEV** [reply]
I have same issue

**Reply by kunalnhes** [reply]
https://github.com/vercel/next.js/discussions/14428#discussioncomment-30169

 View this link have shared a workaround solution for the same.

**Comment by asgaraliyev** [comment, marked as answer]
Try these lines 
```
npm init 
npm install create-next-app@latest --save
npm create-next-app <appname> --use-npm
```

**Reply by raihanhosen011** [reply]
Thanks bro . it's work

**Reply by nsavinda** [reply]
yeah, it's work

**Reply by Franetse7** [reply]
J'ai réussit le travail aussi

**Reply by programmerjeet** [reply]
Thank you, It really works.

**Reply by mayur25-cpuweb** [reply]
what is the issue and can anyone explain this ?

**Reply by ghost** [reply]
This also works based on these steps of commands specified herein https://github.com/vercel/next.js/issues/18188#issuecomment-764923976 as described by https://github.com/adil-waqar .

**Reply by siddharthcode98** [reply]
in the third step instead of `npm create-next-app <appname> --use-npm` I have used `create-next-app <appname> --use-npm` and also I have also installed installed create-next-app globally

**Comment by sojumoscow** [comment]
I have the same problem. I just updated my node and it works!
